### ISSUE TITLE
Missing SHIPPABLE_POSTGRES_VERSION standard environment variable

### ISSUE THREAD

**Issue by kileyhykawy** [Shippable/support#3630]
### Description of your issue:
As part of our CI build, we change the configuration of Postgres so that we can listen on all ports to allow a Docker image running to connect to the database.
```
  ci:
# Reconfigure Postgres to listen and allow same subnet connections so our
# test Docker container can use it as a database
    - echo "listen_addresses = '*'" | sudo tee --append /etc/postgresql/${SHIPPABLE_POSTGRES_VERSION}/main/postgresql.conf >/dev/null
    - echo "host all all samenet trust" | sudo tee --append /etc/postgresql/${SHIPPABLE_POSTGRES_VERSION}/main/pg_hba.conf >/dev/null
    - sudo service postgresql restart
```

 It seems to work on the 'Stable' image but I was trying on 'v5.5.1', the variable is not defined. From the documentation on standard environment variables (http://docs.shippable.com/ci/env-vars/#stdEnv), it should be there?

**Comment by ambarish2012** [comment]
@kileyhykawy thanks for reporting the problem, we are investigating.

**Comment by ambarish2012** [comment]
@kileyhykawy this is indeed a bug, we will let you know as the fix is in production.

**Comment by avinci** [comment]
this was fixed in the latest version deployed yesterday. Please let us know if you still see the issue

**Comment by kanthchandra** [comment]
@avinci @kileyhykawy this will be available when we push a new AMI v5.7.1 in the beginning of next month. So I'm reopening this one and I'll update it as soon as we have the new AMI live in production.

**Comment by ambarish2012** [comment]
@kileyhykawy AMI v5.7.1 has been launched - http://docs.shippable.com/reference/machine-image-v571/.
